stdlib/Random/src/misc.jl

```diff
@@ -343,6 +343,10 @@ Construct a random cyclic permutation of length `n`. The optional `rng`
 argument specifies a random number generator, see [Random Numbers](@ref).
 The element type of the result is the same as the type of `n`.
 
+(Here, a "cyclic permutation" means that all of the elements lie within
```

Comment:
Maybe omit the parens? Given the ambiguity highlighted in the issue, this info seems important enough to be placed in the foreground.

Reply:
Okay.